vividus-to-zephyr-for-jira-exporter/spring-boot-library.gradle

```diff
@@ -0,0 +1,14 @@
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+    ext {
+        springBootVersion = '2.3.0.RELEASE'
+    }
+    dependencies {
+        classpath(group: 'org.springframework.boot', name: 'spring-boot-gradle-plugin', version: springBootVersion)
```

Comment:
could you please add https://docs.spring.io/spring-boot/docs/1.5.16.RELEASE/reference/html/using-boot-devtools.html